apps/site/lib/site_web/templates/page/_tabbed_nav.html.eex

```diff
@@ -0,0 +1,33 @@
+<div class="m-tabbed-nav">
+  <div class="m-tabbed-nav__items">
+    <div class="m-tabbed-nav__item-container">
+      <div class="m-tabbed-nav__item active" data-tab-type="schedules">
+        <span class="m-tabbed-nav__icon notranslate">
+          <%= svg("icon-map-default.svg") %>
+        </span>
+        <span class="m-tabbed-nav__icon-text">Schedules</span>
+      </div>
+    </div>
+    <div class="m-tabbed-nav__item-container">
+      <div class="m-tabbed-nav__item" data-tab-type="trip-planner">
```

Comment:
You've added the `click` event listener to these divs -- so now that they're interactive they need to be focusable. You don't necessarily need to follow [Bootstrap's tab implementation (Dotcom uses v4-alpha.2)](https://getbootstrap.com/docs/4.0/components/navs/#javascript-behavior) but it makes for some reading that does cover a lot of the accessibility concerns that come with this interactivity!

At minimum I'd expect all three tabs to be keyboard navigable.